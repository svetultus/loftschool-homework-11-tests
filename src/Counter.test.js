// Компонент Counter - простой счётчик
// Напишите для него тесты.
// Убедитесь, что вы протестировали всю функциональнось.
// Также проверьте что компонент рендерится верно, используя Snapshot тест.

// * Задание со звёздочкой - выполнять не обязательно

// Вынесите логику в хук и протестируйте его