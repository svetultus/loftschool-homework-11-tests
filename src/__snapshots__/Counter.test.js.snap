// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter соответсвует snapshot'у 1`] = `
Array [
  <div
    className="t-counter"
    data-testid="counter"
  >
    0
  </div>,
  <button
    className="t-btn-increment"
    data-testid="btn-increment"
    onClick={[Function]}
  >
    Increment
  </button>,
  <button
    className="t-btn-decrement"
    data-testid="btn-decrement"
    onClick={[Function]}
  >
    Decrement
  </button>,
]
`;
